clients/src/test/java/org/apache/kafka/clients/admin/AdminClientUnitTestEnv.java

```diff
@@ -71,9 +71,13 @@ public AdminClientUnitTestEnv(Time time, Cluster cluster, Map<String, Object> co
         this.cluster = cluster;
         AdminClientConfig adminClientConfig = new AdminClientConfig(config);
 
+        boolean usingBootstrapController = false;
+        if (config.containsKey(AdminClientConfig.BOOTSTRAP_CONTROLLERS_CONFIG)) {
+            usingBootstrapController = true;
+        }
         AdminMetadataManager metadataManager = new AdminMetadataManager(new LogContext(),
                 adminClientConfig.getLong(AdminClientConfig.RETRY_BACKOFF_MS_CONFIG),
-                adminClientConfig.getLong(AdminClientConfig.METADATA_MAX_AGE_CONFIG), false);
+                adminClientConfig.getLong(AdminClientConfig.METADATA_MAX_AGE_CONFIG), usingBootstrapController);
```

Comment:
we can replace `usingBootstrapController` by `config.containsKey(AdminClientConfig.BOOTSTRAP_CONTROLLERS_CONFIG)` to streamline it

Reply:
Good suggestion. Updated.